letsencrypt/Dockerfile

```
@@ -19,6 +19,7 @@ ARG \
     CERTBOT_DNS_HETZNER_VERSION \
     CERTBOT_DNS_INFOMANIAK_VERSION \
     CERTBOT_DNS_INWX_VERSION \
+    CERTBOT_DNS_JOKER \
```

Seems this is missing `_VERSION`:

```suggestion
    CERTBOT_DNS_JOKER_VERSION \
```